app/views/monitoring.html

```diff
@@ -186,8 +185,8 @@
                       </div>
                       <div class="list-view-pf-additional-info">
                         <div class="list-view-pf-additional-info-item">
-                          <span class="pficon fa-fw pficon-image"></span>
                           <image-names
+                              class="text-prepended-icon word-break"
```

Comment:
Is there any way we can move these classes into the `image-names` view, or do they need to be on this element to work with `pf-list`?